docs/content.zh/docs/deployment/metric_reporters.md

```diff
@@ -63,60 +65,57 @@ metrics.reporter.my_other_reporter.host: 192.168.1.1
 metrics.reporter.my_other_reporter.port: 10000
 ```
 
-**Important:** The jar containing the reporter must be accessible when Flink is started. Reporters that support the
- `factory.class` property can be loaded as [plugins]({{< ref "docs/deployment/filesystems/plugins" >}}). Otherwise the jar must be placed
- in the /lib folder. Reporters that are shipped with Flink (i.e., all reporters documented on this page) are available
- by default.
+**注意**：Flink 在启动时必须能访问到发送器所属的 jar 包。支持 `factory.class` 配置的发送器可以被加载为 [plugins]({{< ref "zh/docs/deployment/filesystems/plugins" >}})，没有该项配置的发送器需要保证 jar 包在 /lib 目录下。Flink 自带的发送器（文档中已经列出的发送器）无需做其他配置，开箱即用。
+
+你可以实现 `org.apache.flink.metrics.reporter.MetricReporter` 接口来自定义发送器，并实现 `Scheduled` 接口让发送器周期性地将运行时指标发送出去。
+另外也可以实现 `MetricReporterFactory` 接口，让发送器作为插件被 Flink 导入。
 
-You can write your own `Reporter` by implementing the `org.apache.flink.metrics.reporter.MetricReporter` interface.
-If the Reporter should send out reports regularly you have to implement the `Scheduled` interface as well.
-By additionally implementing a `MetricReporterFactory` your reporter can also be loaded as a plugin.
+接下来的部分列出了 Flink 支持的采集器。
 
-The following sections list the supported reporters.
+<a name="jmx"></a>
 
-### JMX 
+### JMX
 #### (org.apache.flink.metrics.jmx.JMXReporter)
 
-You don't have to include an additional dependency since the JMX reporter is available by default
-but not activated.
+JMX 发送器默认可直接使用，无需引入其他依赖。
 
-Parameters:
+参数：
 
-- `port` - (optional) the port on which JMX listens for connections.
-In order to be able to run several instances of the reporter on one host (e.g. when one TaskManager is colocated with the JobManager) it is advisable to use a port range like `9250-9260`.
-When a range is specified the actual port is shown in the relevant job or task manager log.
-If this setting is set Flink will start an extra JMX connector for the given port/range.
-Metrics are always available on the default local JMX interface.
+- `port` - (非必须) JMX 监听的端口。
```

Comment:
`(`->`）`

please also change in the rest of the page.

Reply:
Ok,I will fix this,thanks for your review work!

Reply:
Fixed.